### DISCUSSION TITLE
Balancing the start of the game

### DISCUSSION THREAD

**Discussion by Dionis404** [sunflower-land/sunflower-land#4891] [General]
Here will be collected my observations and thoughts on how the start of the game is currently implemented and what can be changed for more player engagement. The base island is a tutorial and the main idea is that the player would pass this stage more quickly.

In the first few minutes of the game, the player unlocks 3 expansions 
Opening the next expansion is a longer process, and it feels like a dramatic increase in the game's complexity.
Requires level 5 and 1 iron
![image](https://github.com/user-attachments/assets/c347d71f-701a-4db0-a2c0-efd1475f06d4)

### Building a well.
It takes a lot of time to build a well.
And new players often have a question how many beds opens a well.
![image](https://github.com/user-attachments/assets/d84cbd25-a070-4990-bfa1-9e8180fcbde4)

Solution: make the first well cheaper or free.
Add more hints about the operation of the well. 

### **There are too many dishes available to cook**
![image](https://github.com/user-attachments/assets/c2fa024f-6757-4965-8a21-db7814ff04d9)
This can be very confusing for the player and forces them to learn a lot of information at the start of the game

Solution: Limit the ability to cook certain dishes. 

### Wood, stone, iron and gold.
![image](https://github.com/user-attachments/assets/de5d3354-7d7b-4ee7-998a-40c09580cf0b)

Iron and gold are difficult to mine for a new player takes a lot of time. 
Solution options:
Open these resources on later extensions.
Give the player a starter toolkit?

### The main purpose of this topic is to offer the best possible training option for new players

The topic will be added to over time.

**Comment by TheDcol** [comment]
Thanks Dionis, having gone through the onboarding process numerous times I can agree with most of what you said. You would have noticed the requirements got significantly reduces both from levelling perspective and resource reqs for expanding compared to 6~ months ago. Afew more tweaks might be required for that first hour or 2 of gameplay to progress.

Do you think the crops should unlock a bit later or do you think most of the crops being unlocked early is ok?

**Reply by Dionis404** [reply]
Yeah, I've had thoughts of confining most of the cultures to the base island.

**Comment by Dionis404** [comment]
### 10 Training Steps
The idea is that the basic island becomes a complete training for the player.
This is necessary so that the player gets into the game faster and can develop his farm.
At the moment, players on the basic island need about a week without investment to pass this stage. This is a very boring process that does not help the player to develop.  Below I will try to describe each step of the training and the changes that would be worth making






## Step 1: Welcome

After creating an account, a welcome screen appears in front of the player.

Pete tells the player where he disappeared to and introduces him to the mayor.

The mayor asks the player's name.

[The player enters his name.]

Pete talks about the first steps and introduces the player to the blacksmith's craft.

[The forge appears on the farm.]

The blacksmith talks about buying axes and asks to cut down a tree, giving the initial number of axes.

[The player cuts down the tree.]

Pete talks about the possibility of expanding the farm.

[The player opens a new expansion.]

[The beds are open.]

(4 EXPANSIONS)


## Step 2: Landings

Pete introduces Betty.

[The farm store appears.]

Betty talks about buying seeds and selling the crops.

Pete talks about the sunflowers growing in the beds.

[Player picks sunflowers.]

Betty says she'll buy the sunflowers for double the price.

[x2 sale price before moving to the new island.]

## Step 3: Scarecrow

The blacksmith tells the player that he knows how to speed up the growth of sunflowers and other basic crops, and says that the player will need wood to create the Scarecrow

[Player mines the required amount of resources]

[Player crafts a scarecrow]

[Player installs scarecrow]

Betty tells that the scarecrow only works in the area in front of you, and cannot be moved while the crops are growing.

## Step 4: Food.

Pete says to open more expansions

[Player gathers resources to open]

[Player opens an expansion]

The Fire Pete appears.

The player is told about cooking and asked to cook food.

[Player cooks food]

[Player eats food]

Player's level increases

(5 EXPANSIONS)

## Step 5: Skills



Step 6:



Step 7:



Step 8:



Step 9:



Step 10:
